src/components/MultiLink/Legend.vue

```diff
@@ -58,6 +59,45 @@ export default Vue.extend({
     nodeColorScale() {
       return store.getters.nodeColorScale;
     },
+
+    columnTypes() {
+      const typeMapping: { [key: string]: string} = {};
+
+      if (store.getters.networkMetadata !== null) {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        Object.entries(store.getters.networkMetadata).forEach(([tableName, metadata]) => {
```

Comment:
According to these [TS docs](https://www.typescriptlang.org/docs/handbook/variable-declarations.html#array-destructuring), you can write this as:

```suggestion
        Object.entries(store.getters.networkMetadata).forEach(([, metadata]) => {
```

It's slightly ugly, but if this actually works (and the linter is ok with it) it's actually a better choice, because it prevents any errors from developing around the unused `tableName` variable.

Reply:
Actually, if all you want is the object values, you should be able to get them directly with `Object.values()`, instead of calling `.entries()` and throwing away the first half of each element.

Reply:
Absolutely! `.entries()` is a much better way to do this

Reply:
Fixed in c176361

Reply:
You should remove the comment directive.